azure-proto-compute/Extensions/ResourceGroupExtensions.cs

```diff
@@ -9,53 +9,53 @@ public static class ResourceGroupExtensions
     {
         #region VirtualMachines
         /// <summary>
-        /// Gets the VirtualMachineOperations for a VirtualMachine.
+        /// Gets an object representing the operations that can be performed over a specific <see cref="[VirtualMachine]" />.
```

Comment:
the cref should be VirtualMachineOperations since thats what is being returned, or it should simply just have text VirtualMachine.  I would lean towards just text since the type is already in the response statement